src/modules/generateRandomNumber.js

```diff
@@ -7,7 +7,19 @@
  * @return {number} A random 4-digit number
  */
 function generateRandomNumber() {
-  /* Write your code here */
+  const digits = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const vailableDigits = [...digits];
+  let number = '';
+
+  for (let i = 0; i < 4; i++) {
+    const randomIndex = Math.floor(Math.random() * vailableDigits.length);
```

Comment:
Ensure that the typo in `vailableDigits` is corrected to `availableDigits` here as well.